### PR TITLE
fix(client): rename certification for typo

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Either, going to refactor the code to not have hard-coded values, or leave as is (remove comment), and accept just rename.

Edit; just leave as is, and fix typo

### CODE REVIEW

File: client/src/components/settings/Certification.js [hunk 1]
```diff
@@ -508,7 +508,7 @@ export class CertificationSettings extends Component {
         ? verifyCert(superBlock)
         : createFlashMessage(honestyInfoMessage);
     };
-
+    // TODO: refactor to not have hardcoded certificates
```
Comment (ojeytonwilliams) [inline client/src/components/settings/Certification.js:511]:
```suggestion
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Could you remove the comment?

**Review by RandellDawson — APPROVED** [review]
LGTM

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**